src/obr/core/core.py

```diff
@@ -142,6 +142,32 @@ def logged_func(func, doc, **kwargs):
     doc["history"].append(res)
 
 
+def get_mesh_stats(owner_path: str) -> dict:
+    """Check constant/polyMesh/owner file for mesh properties
+    and return it via a dictionary"""
+    nCells = None
+    nFaces = None
+    if Path(owner_path).exists():
```

Comment:
```suggestion
    if Path(owner_path).exists():
            found_note = ""
```

Reply:
This might be poorly intended by me. My reason for pulling this out of the loop is because it is accessed in an outer scope of its definition, which somehow works in python, _but_ I would consider to be bad practise.